builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -94,6 +124,15 @@ func resourceAwsS3BucketRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	// Add website_endpoint as an output
+	endpoint, err := websiteEndpoint(s3conn, d)
+	if err != nil {
+		return err
+	}
+	if err := d.Set("website_endpoint", endpoint); err != nil {
+		return err
+	}
```

Comment:
I'd actually just add an acceptance test that includes the disable case to test this. So it'd be a test with one step that has website support and the next step that removes it.